components/infra-proxy-service/server/orgs_test.go

```diff
@@ -1031,7 +1042,7 @@ func TestOrgs(t *testing.T) {
 				Projects:  []string{},
 			})
 			require.Nil(t, resp)
-			assert.Contains(t, err.Error(), "must supply org ID")
+			assert.Error(t, err, "must supply org ID")
```

Comment:
Loosely coupled assert to check the Err object only, avoiding hardcoded error message check here, any thoughts would be helpful!